src/Requests/EncryptDecryptRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Keboola\AzureKeyVaultClient\Requests;
+
+use Keboola\AzureKeyVaultClient\Exception\ClientException;
+
+class EncryptDecryptRequest
+{
+    const RSA_OAEP = 'RSA-OAEP';
+    const RSA_OAEP_256 = 'RSA-OAEP-256';
+    const RSA_1_5 = 'RSA1_5';
```

Comment:
Trochu mimo tohto review, ale vies aky je rozdiel medzi tymito sifrovacimi algoritmami? Ja som sa to snazil googlit trochu, ale nic moc som nenasiel.
Chcel by som vediet, ktory pouzit v OAuth API :)

Reply:
rekl bych, ze RSA-OAEP-256, RSA1_5 je discouraged https://connect2id.com/products/nimbus-jose-jwt/examples/jwt-with-rsa-encryption a 256 je max delka klice

Reply:
Ok, diky